lib/main_containers.pm

```diff
@@ -0,0 +1,92 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: module loader of container tests
+# Maintainer: qa-c@suse.de
+
+package main_containers;
+use base 'Exporter';
+use Exporter;
+use utils;
+use version_utils;
+use main_common qw(loadtest boot_hdd_image);
+use testapi qw(check_var get_required_var get_var);
+use Utils::Architectures;
+use strict;
+use warnings;
+
+
+our @EXPORT = qw(
+  is_container_test
+  load_container_tests
+);
+
+sub is_container_test {
+    get_var('CONTAINER_RUNTIME') ? return 1 : return 0;
+}
+
+sub load_tests_podman_image {
+    loadtest 'containers/podman_image';
+}
+
+sub load_tests_docker_image {
+    loadtest 'containers/docker_image';
+    loadtest 'containers/container_diff';
+}
+
+sub load_tests_podman_common {
+    unless (is_sle('<=15')) {
```

Comment:
I know `unless` is cool, but maybe if is more readable
```suggestion
    if (is_sle('>15')) {
```

Reply:
ok